### DISCUSSION TITLE
"Go Back" button

### DISCUSSION THREAD

**Discussion by xiaoniaoawa** [ppy/osu#29873] [Ideas] [answered]
I jump between many page, but I can not go back, maybe can make a button to go back?

![image](https://github.com/user-attachments/assets/8b70a27f-8d8e-4f2f-a337-2363b2d9dd44)
for example, place a back button to the cursor point in the image to go the pervous page may pretty useful in sometimes(?

**Comment by Joehuu** [comment, marked as answer]
Already tracked as an issue: https://github.com/ppy/osu/issues/5063

**Reply by xiaoniaoawa** [reply]
Ok thanks
